scripts/aerosol_performance/runThisGPU.sh

```diff
@@ -20,6 +20,7 @@ for N in ${nbatch[@]}; do
                --verbose=$verbose \
                --outputfile_times=$output_wall_times \
                --outputfile=$output_file \
+               --number_of_particles=$number_of_particles \
```

Comment:
@samfrederick, here's the reason the number of particles didn't impact the GPU's performance. I omitted this parameter from the command line, resulting in all our runs defaulting to a single particle.